sympy/physics/mechanics/kane.py

```diff
@@ -923,36 +925,57 @@ def kanes_equations(self, bodies, loads=None):
 
         return (self._fr, self._frstar)
 
+    def _kanes_equations_not_called(self, attr):
+        if not self._fr or not self._frstar:
+            msg = 'kanes_equations() must be run before {} is available.'
+            raise AttributeError(msg.format(attr))
+
     def rhs(self, inv_method=None):
-        """Returns the system's equations of motion in first order form.
+        """Returns the right hand side of the explicit first order full
+        differential equations of motion where:
```

Comment:
"full differential equations of motion ~~where~~:" I think this would flow better.